recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/guest_doma

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+#
+# xencommons    Script to start DomA guest
+#
+# Author:       Andrii Anisov <andrii_anisov@epam.com>
+#
+# chkconfig: 2345 70 10
+# description: Starts and stops xenstored and xenconsoled
+### BEGIN INIT INFO
+# Provides:          guest_doma
+# Required-Start:    $guest_domd
+# Should-Start:
+# Required-Stop:     $guest_domd
+# Should-Stop:
+# Default-Start:     2 3 5
+# Default-Stop:      0 1 6
+# Short-Description: Start/stop DomA
+# Description:       Simple run of DomA, will be updated soon
+### END INIT INFO
+
+# not running in Xen dom0 or domU
+if ! test -d /proc/xen ; then
+	exit 0
+fi
+
+do_start () {
+    echo "Starting DomA"
```

Comment:
you can define DomA as a variable and reference through the script
this will also allow generalizing the script and have a single place where you
change variables